pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[tool.poetry]
+name = "manim"
```

Comment:
DO we have the `manim` name? Should I need to change it to something else?

Reply:
I am changing this to `manimlib` for two reasons
- We still didn't got the name manim
- The chocolatey package which was approved yesterday requires this. So I think this should be the last commit we make for release to prevent it from breaking. After releasing I will make a new release for chocolatey which will fix this.

Reply:
Can you please make sure to add this as part of the v1.0.0 release milestone so we don't forget to do this?